sharding-core/src/test/java/io/shardingsphere/core/keygen/KeyGeneratorFactoryTest.java

```diff
@@ -55,6 +63,10 @@ public void assertCreateKeyGeneratorFailureWithIllegalAccess() {
     
     @NoArgsConstructor(access = AccessLevel.PRIVATE)
     public static final class IllegalAccessKeyGenerator implements KeyGenerator {
+    
+        @Getter
+        @Setter
+        private Properties keyGeneratorProperties = new Properties();
```

Comment:
Move `@Getter` and `@Setter` to class level if all fields need getter and setter, even there is only one field.